static/swaggerui/swagger.json

```diff
@@ -0,0 +1,456 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "This is the first version of Kube-green API.",
+    "version": "1.0.0",
+    "title": "Kube-green API",
+    "termsOfService": "http://swagger.io/terms/",
+    "contact": {
+      "email": "apiteam@swagger.io"
+    },
+    "license": {
+      "name": "Apache 2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  },
+  "host": "",
+  "basePath": "/v1",
+  "tags": [
+    {
+      "name": "kube-green",
+      "description": "Make your k8s clusters more green",
+      "externalDocs": {
+        "description": "Find out more",
+        "url": "https://mia-platform.eu/"
+      }
+    }
+  ],
+  "schemes": [
+    "https",
+    "http"
+  ],
+  "paths": {
+    "/cpu_usage": {
+        "summary": "CPU usage",
+        "description": "Get CPU usage",
+        "operationId": "getCpuUsage",
+        "produces": [
+          "application/xml",
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "namespace",
+            "in": "query",
+            "description": "name of namespace",
+            "required": true,
```

Comment:
Namespace is not required, can be empty, which will return all namespaces, we may need to improve the documentation too. Please fix the required parameter for all